### DISCUSSION TITLE
【参考文献格式】当只引用了两篇参考文献时，如何设置这两篇文献的序号之间用逗号，而不是用“-”？

### DISCUSSION THREAD

**Discussion by ChaiHuichao** [tuna/thuthesis#974] [Q&A] [answered]
您好，如在正文中使用\cite{A, B}命令引用两篇文献，则希望在生成的PDF中对应处上标为[1,2]，当引用大于等于3篇参考文献时，对应上标才采用"-"连接，如[1-3]。而现在的模板中只要引用大于等于2篇参考文献，则均用"-"连接，该怎么修改呢？

**Comment by zepinglee** [comment]
GB/T 7714—2015 要求的。

<img width="793" alt="Screenshot 2024-08-20 at 23 46 04" src="https://github.com/user-attachments/assets/18552546-8e02-4150-96bf-5a0a23401208">

**Comment by ChaiHuichao** [comment]
感谢回复，这个问题主要是匿名评审人提出了要求，说当只引用了两篇参考文献时，这两篇文献的序号之间应该用逗号，而不是用“-”，所以想着按他的要求改了就行了，不然解释起来也很麻烦。所以想请教一下您按这个要求来排参考文献，该怎么修改呢？

**Reply by zepinglee** [reply, marked as answer]
https://github.com/tuna/thuthesis/blob/60661f7d140c192418cc9151b54a6c59a99fae23/thuthesis.dtx#L5760-L5768

这一段将修改后的 5768 行替换为 `natbib` 原来的内容（\#5763–\#5767 注释掉的部分）。

顺便指出学校的《写作指南》第 20 页也是要求的连续两个编号用短横线。吐槽一下这么多评审人都有自己的想法。

<img width="732" alt="Screenshot 2024-08-21 at 10 34 59" src="https://github.com/user-attachments/assets/f2d2538b-0498-439e-8857-663ec8d38140">

**Reply by ChaiHuichao** [reply]
再次感谢回复，问题已顺利解决。评审人的多样性确实是做论文模板以及写论文的一大难题。
